website/docs/overrides/power-select/partials/version-history/4.3.0.md

```diff
@@ -0,0 +1,9 @@
+## 4.3.0
+
+### Deprecated
+
+In this version we have released the [SuperSelect](/components/form/super-select) component, as replacement for the `PowerSelect` Ember addon component. For this reason we now consider the PowerSelect style overrides as deprecated and will be removed in a future major release.
```

Comment:
question: should we also update [our wiki docs](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#breaking-changes) to formalize how a deprecation message should look like?

Reply:
My thoughts, while writing this text, were two
1. we need some time to collect a few use cases and see if there are emerging patterns we can formalize
2. this gives us the freedom to write different content for different cases, depending on the context or what is required by the consumers to do

Reply:
makes sense!